packages/storage/amplify_storage_s3/example/integration_test/main_test.dart

```diff
@@ -262,6 +265,7 @@ void main() async {
                   metadata: {
                     'filename': testObjectFileName3,
                   },
+                  useAcceleration: true,
```

Comment:
question: Do we also need same test with accel set to `false` to test to keep coverage for use without it?

Reply:
The default value is false, the tests are not using `useAcceleration: true` should cover the case `useAcceleration: false`.

Reply:
Added dedicated group to test.